config/checkstyle/tree_walker_module_directives.xml

```diff
@@ -44,9 +44,9 @@
 </module>
 <module name="MissingSwitchDefault"/>
 <module name="MultipleStringLiterals">
-    <property name="allowedDuplicates" value="3"/>
+    <property name="allowedDuplicates" value="5"/>
     <!-- Allow for more than 3 occurrences of "" or ", " string in the given file -->
-    <property name="ignoreStringsRegexp" value="^(&quot;&quot;|&quot;, &quot;)$"/>
+    <property name="ignoreStringsRegexp" value="^(&quot;&quot;|&quot;, &quot;| )$"/>
```

Comment:
```suggestion
    <property name="ignoreStringsRegexp" value="^&quot;(| |, )&quot;$"/>
```

Reply:
The new alternative has been ignored since it didn't including the leading&trailing `"` (maybe that was the reason why 5 duplicates were needed?)